plugin/src/main/kotlin/com/slapin/gsp/internal/api/SamsungApiClient.kt

```diff
@@ -107,11 +107,12 @@ internal class SamsungApiClient(
                 contentId = currentContentInfo.contentId,
                 defaultLanguageCode = currentContentInfo.defaultLanguageCode,
                 paid = currentContentInfo.paid,
-                usExportLaws = true,
+                usExportLaws = currentContentInfo.usExportLaws,
                 ageLimit = currentContentInfo.ageLimit,
                 binaryList =
                   listOf(
                     Binary(
+                      binarySeq = 1,
```

Comment:
It was also intentional as during the release cycle there could be more than 1 binaries uploaded to the same publication and setting this will break the ability to do so.